public/src/components/amounts/AmountsTestEditor.tsx

```diff
@@ -357,6 +361,14 @@ export const AmountsTestEditor: React.FC<AmountsTestEditorProps> = ({
           </>
         )}
 
+        {!checkIfTestIsCountryTier() && (
+          <Typography className={classes.note}>
+            Note: users arriving at the Supporter+ landing page from Apple News/Google AMP article
```

Comment:
My preference would be for AN users to only see the control variant. We AB test nothing else for AN. I think we only give AMP users the control variant in any case because I haven't touched AMP files in any of my amounts testing work!

Reply:
Perhaps a question for Marketing. There's no reason we can't AB test amounts for users from AN on the checkout - and we currently do

Reply:
Done!